docs/apis/ou.yaml

```diff
@@ -0,0 +1,262 @@
+                  value:
+                    id: "4d4cb83a-9bb8-4fd7-970e-36b65f9c8ea4"
+                    name: "Frontend Team"
+                    description: "Handles UI/UX work"
+                    parent: "afc77cfd-620b-4cf0-a31c-7377b0ea8902"
+
+  /organization-units/{id}:
+    get:
+      tags:
+        - organization-units
+      summary: Get a organization unit by id
+      parameters:
+        - in: path
+          name: id
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        '200':
+          description: Organization unit details
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OrganizationUnit'
+              example:
+                id: "afc77cfd-620b-4cf0-a31c-7377b0ea8902"
+                name: "Engineering Team"
+                description: "Engineering unit that handles all engineering tasks"
+                parent: null
+                users:
+                  - "3f9d5c8e-3c56-4f09-a1db-bb3d5e3f4b5d"
+                  - "9f4bc2ee-8d1e-4e21-a3b4-17d9b612e320"
+                  - "20eb225a-e8c3-4ccf-9d47-b659e26014d7"
+                groups:
+                  - "c2a4f62d-7c79-46f2-b2de-8e8cb4f1c456"
+                  - "8a8e4fa2-2e55-478d-bf0f-4a2ddaf9ee7d"
+                sub-organization-units:
+                  - "6b59f277-fd79-40b1-afca-dce75d3709f7"
+        '404':
+          description: Organization unit not found
+
+    put:
+      tags:
+        - organization-units
+      summary: Update an organization unit by id
+      parameters:
+        - in: path
+          name: id
+          required: true
+          schema:
+            type: string
+            format: uuid
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/UpdateOrganizationUnitRequest'
+            example:
+              name: "Engineering Team"
+              description: "Engineering unit that handles all engineering tasks"
+              parent: null
+      responses:
+        '200':
+          description: Organization unit updated
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OrganizationUnit'
+              example:
+                id: "afc77cfd-620b-4cf0-a31c-7377b0ea8902"
+                name: "Engineering Team"
+                description: "Engineering unit that handles all engineering tasks"
+                parent: null
+                users:
+                  - "3f9d5c8e-3c56-4f09-a1db-bb3d5e3f4b5d"
+                  - "9f4bc2ee-8d1e-4e21-a3b4-17d9b612e320"
+                  - "20eb225a-e8c3-4ccf-9d47-b659e26014d7"
+                groups:
+                  - "c2a4f62d-7c79-46f2-b2de-8e8cb4f1c456"
+                  - "8a8e4fa2-2e55-478d-bf0f-4a2ddaf9ee7d"
```

Comment:
Same as previous comment. Do we need to return users and groups?

Reply:
Lets discuss and conclude.